core/src/mindustry/logic/LStatements.java

```diff
@@ -1291,16 +1291,21 @@ public LCategory category(){
 
     @RegisterStatement("spawnwave")
     public static class SpawnWaveStatement extends LStatement{
+        public String natural = "false";
```

Comment:
Would be more backwards-compatible to have this as a last parameter so existing logic doesn't break.